services/pypi/pypi-typing.service.js

```diff
@@ -0,0 +1,40 @@
+import PypiBase, { pypiGeneralParams } from './pypi-base.js'
+
+export default class PypiTyping extends PypiBase {
+  static category = 'platform-support'
+
+  static route = this.buildRoute('pypi/typing')
```

Comment:
For consistency, the closest other badge to this is the typescript badge, which uses "types" (as opposed to "typing") as the noun part of the route and label. Can we go with `types` here too for both the route and label.